docs/kcl_mod.md

```diff
@@ -0,0 +1,70 @@
+# kcl.mod
+
+## Background
+kpm needs a configuration file for KCL project, which is responsible for marking the root directory, determining dependency versions, configuring compilation options or others.
```

Comment:
You can use https://github.com/KusionStack/KCLVM/issues/339 and https://github.com/KusionStack/KCLVM/issues/338 in `Background`.

Reply:
I removed the backgroud in this doc, backgroud should probably be in the README on the front page. This is a kcl.mod spec.